samples/CommunityToolkit.Maui.Sample/ViewModels/Layouts/UniformItemsLayoutViewModel.cs

```diff
@@ -12,5 +12,12 @@ void AddItem()
 		Items.Add(Path.GetRandomFileName());
 	}
 
+ 	[RelayCommand]
+	void ClearItem()
+	{
+		Items = null;
+		OnPropertyChanged(nameof(Items));
```

Comment:
Perhaps we want to just clear this or set to a new instance to avoid `AddItem` from crashing.
```suggestion
		Items.Clear();
```